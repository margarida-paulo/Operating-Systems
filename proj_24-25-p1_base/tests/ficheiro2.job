HELP
SHOW
WRITE [(NOME,VALOR)(NOME2,VALOR2)]
SHOW
