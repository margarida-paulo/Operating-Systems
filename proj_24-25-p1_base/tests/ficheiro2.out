(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolsty)
(Luís, Camões)
(Marcel, Proust)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
Available commands:
  WRITE [(key,value)(key2,value2),...]
  READ [key,key2,...]
  DELETE [key,key2,...]
  SHOW
  WAIT <delay_ms>
  BACKUP
  HELP
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolsty)
(Luís, Camões)
(Marcel, Proust)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Isabel, Allende)
(Italo, Calvino)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Leo, Tolsty)
(Luís, Camões)
(Mariana, Mari)
(Marcel, Proust)
(Miguel, Cervantes)
(Oscar, Wilde)
(Pablo, Neruda)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(William, Faulkner)
(Zadie, Smith)
((AlbertAlbert, , CamusCamus)
)
((AldousAldous, , HuxleyHuxley)
)
((AgathaAgatha, , ChristieChristie)
)
((ChinuaChinua, , AchebeAchebe)
)
((C.S.C.S., , LewisLewis)
)
((CharlesCharles, , DickensDickens)
)
((ClariceClarice, , LispectorLispector)
)
((ErnestErnest, , HemingwayHemingway)
)
((EdgarEdgar, , AllanAllan)
)
((EmilyEmily, , DickinsonDickinson)
)
((FyodorFyodor, , DostoevskyDostoevsky)
)
((FranzFranz, , KafkaKafka)
)
((FernandoFernando, , PessoaPessoa)
)
((GeorgeGeorge, , OrwellOrwell)
)
((GabrielGabriel, , MarquezMarquez)
)
((HarukiHaruki, , MurakamiMurakami)
)
((IsabelIsabel, , AllendeAllende)
)
((ItaloItalo, , CalvinoCalvino)
)
((JamesJames, , JoyceJoyce)
)
((J.R.R.J.R.R., , TolkienTolkien)
)
((J.K.J.K., , RowlingRowling)
)
((JaneJane, , AustenAusten)
)
((JoséJosé, Saramago, Saramago)
()
(Jorge, Jorge, Luis)
Luis)
(Leo(Leo, Tolsty, Tolsty)
()
(Luís, Luís, Camões)
Camões)
(Margarida(Margarida, Maggie, Maggie)
()
(Mariana, Mariana, Mari)
Mari)
(Marcel(Marcel, Proust, Proust)
()
(Miguel, Miguel, Cervantes)
Cervantes)
(Oscar(Oscar, Wilde, Wilde)
()
(Pablo, Pablo, Neruda)
Neruda)
(Simone(, OliveiraSimone, )
(Oliveira)
Stephen, (StephenKing)
, King(Sophia)
(, AndresenSophia, )
(Andresen)
Toni, (ToniMorrison)
, Morrison(Victor)
(, HugoVictor, )
(Hugo)
Virginia, (VirginiaWoolf)
, Woolf(William)
(, FaulknerWilliam, )
(Faulkner)
Zadie, (ZadieSmith)
, Smith(Albert)
(, CamusAlbert, )
(Camus)
Aldous, (AldousHuxley)
(, HuxleyAgatha, )
(Christie)
Agatha, (ChinuaChristie)
, Achebe(Chinua)
(, AchebeC.S., )
(Lewis)
C.S., (CharlesLewis)
, Dickens(Charles)
(, Dickens)
Clarice, (ClariceLispector)
, Lispector(Ernest)
(, HemingwayErnest, )
(Hemingway)
Edgar, (EdgarAllan)
, Allan(Emily)
(, DickinsonEmily, )
(Dickinson)
Fyodor, (FyodorDostoevsky)
, Dostoevsky(Franz)
(, KafkaFranz, )
(Kafka)
Fernando, (FernandoPessoa)
, Pessoa(George)
(, OrwellGeorge, )
(Orwell)
Gabriel, Marquez(Gabriel)
(Haruki, Marquez, Murakami)
()
(Haruki, MurakamiIsabel, )
(Allende)
Isabel, Allende(Italo)
(, CalvinoItalo, )
(Calvino)
James, (JamesJoyce)
, Joyce(J.R.R.)
(, TolkienJ.R.R., )
(Tolkien)
J.K., (J.K.Rowling)
, Rowling(Jane)
(, AustenJane, )
(Austen)
José, (JoséSaramago)
, Saramago(Jorge)
(, LuisJorge, )
(Luis)
Leo, (LeoTolsty)
, Tolsty(Luís)
(, CamõesLuís, )
(Camões)
Margarida, (MargaridaMaggie)
, Maggie(Mariana)
(, MariMariana, )
(Mari)
Marcel, (MarcelProust)
, Proust(Miguel)
(, CervantesMiguel, )
(Cervantes)
Oscar, (OscarWilde)
, Wilde(Pablo)
(, NerudaPablo, )
(Neruda)
Simone, (SimoneOliveira)
, Oliveira(Stephen)
(, KingStephen, )
(King)
Sophia, (SophiaAndresen)
, Andresen(Toni)
(, MorrisonToni, )
(Morrison)
Victor, (VictorHugo)
, Hugo(Virginia)
(, WoolfVirginia, )
(Woolf)
William, (WilliamFaulkner)
, Faulkner(Zadie)
(, SmithZadie, )
Smith)
Available commands:
  WRITE [(key,value)(key2,value2),...]
  READ [key,key2,...]
  DELETE [key,key2,...]
  SHOW
  WAIT <delay_ms>
  BACKUP
  HELP
Available commands:
  WRITE [(key,value)(key2,value2),...]
  READ [key,key2,...]
  DELETE [key,key2,...]
  SHOW
  WAIT <delay_ms>
  BACKUP
  HELP
Available commands:
  WRITE [(key,value)(key2,value2),...]
  READ [key,key2,...]
  DELETE [key,key2,...]
  SHOW
  WAIT <delay_ms>
  BACKUP
  HELP
